src/runtime/parser.c

```diff
@@ -839,7 +839,7 @@ static void parser__accept(Parser *self, StackVersion version,
 
     if (parser__select_tree(self, self->finished_tree, root)) {
       ts_tree_release(self->finished_tree);
-      assert(root->ref_count > 0);
+      assert((root == NULL) || (root->ref_count > 0));
```

Comment:
`root` should never be `NULL` here; would it make `scan-build` happy if we changed this to this?

```c
assert(root && root->ref_count > 0);
```